05-bots/README.md

```diff
@@ -56,6 +56,7 @@ Prepare a 5 minute presentation on your bot on October 16.
 _(Please note you are welcome to post under a pseudonym and/or password protect your published assignment. For NYU blogs, privacy options are covered in the [NYU Wordpress Knowledge Base](https://wp.nyu.edu/knowledge/). Finally, if you prefer not to post your assignment at all here, you may email the submission.)_
 
 - Name - [project title](url)
+- LovelyA72 - [Week 5](https://a2z.kmoene.com/p/073722-0903.html)
```

Comment:
```suggestion
- LovelyA72 - [AMLT The Chat Bot](https://a2z.kmoene.com/p/073722-0903.html)
```

I prefer the links to have a descriptive title. This is best for accessibility reasons! How is this?